doc/JFLA2022/paper.tex

```diff
@@ -150,7 +151,8 @@ \subsection{Background}
 \end{itemize}
 
 \subsection{Vision}
-The \emph{Hydra-battles} project, part of \community~\cite{CoqCommunity} aims to be an experimental platform for the collaborative development of commented libraries of formal proofs. \community is a community organization that we have founded in 2018 with two goals in mind: providing a solution for the long-term maintenance of interesting \coq packages, and working collaboratively on documentation projects. The \emph{Hydra-battles} project demonstrates that these two goals are not independent: interesting \coq packages can become the basis for new documentation.
+The \Hydras project, part of \Coqcommunity\cite{CoqCommunity} aims to be an experimental platform for the collaborative development of commented libraries of formal proofs. \Coqcommunity is a community organization that we have founded in 2018 with two goals in mind: providing a solution for the long-term maintenance of interesting Coq packages, and working collaboratively on documentation projects. The \Hydras project demonstrates that these two goals are not independent: interesting Coq packages can become the basis for new documentation.

```

Comment:
```suggestion
The \Hydras project, part of \Coqcommunity~\cite{CoqCommunity} aims to be an experimental platform for the collaborative development of commented libraries of formal proofs. \Coqcommunity is a community organization that we have founded in 2018 with two goals in mind: providing a solution for the long-term maintenance of interesting Coq packages, and working collaboratively on documentation projects. The \Hydras project demonstrates that these two goals are not independent: interesting Coq packages can become the basis for new documentation.
```
If you apply the fix to `\Coqcommunity` macro, we should be able to use the standard `~\cite{XY}`.